acquia_lift.routing.yml

```diff
@@ -4,4 +4,47 @@ acquia_lift.admin_settings_form:
     _form: '\Drupal\acquia_lift\Form\AdminSettingsForm'
     _title: 'Acquia Lift settings'
   requirements:
-    _permission: 'administer acquia lift'
+    _permission: 'administer acquia_lift'
+
+acquia_lift.slot.overview:
+  path: '/admin/config/content/acquia-lift/slots'
+  defaults:
+    _title: 'Slots'
+    _entity_list: 'acquia_lift_slot'
+  requirements:
+    _entity_create_access: 'acquia_lift_slot'
+
+entity.acquia_lift_slot.add_form:
+  path: '/admin/config/content/acquia-lift/slots/add-facet'
```

Comment:
/admin/config/content/acquia-lift/slots/add then, same reasoning. /admin/config/content/acquia-lift/slot does not exist, so I don't feel comfortable using a route where the parent does not correspond to anything.